templates/docs/upgrade-guide-v3.md

```diff
@@ -157,6 +157,10 @@ The label pattern has been removed, and we've added tinted chips. The tints are
 
 The `vf-p-label` mixin has been removed so you'll have to remove this from your code, along with `vf-p-label-new`, `vf-p-label-updated`, `vf-p-label-deprecated`, `vf-p-label-in-progress` and `vf-p-label-validated`.
 
+## Chips
+
+By default chips are not interactive anymore (they don't have hover, focus or active state). To make chip interactive you should use `<button class="p-chip">` instead of `<span class="p-chip">`.
```

Comment:
```suggestion
By default, chips are not interactive anymore (they don't have hover, focus or active state). Apply the class to a button element to get interactive behaviour, e.g.: `<button class="p-chip">` instead of `<span class="p-chip">`.
```